daemon/graphdriver/lcow/lcow.go

```diff
@@ -2,8 +2,22 @@
 
 package lcow
 
-// Maintainer: @jhowardmsft
-// Graph-driver for Linux Containers On Windows (LCOW)
+// Maintainer:	jhowardmsft
+// Locale:		en-gb
+// About: 		Graph-driver for Linux Containers On Windows (LCOW)
+//
+// This graphdriver runs in two modes. Yet to be determined which one will
+// be the shipping mode. The global mode is where a single utility VM
+// is used for all service VM tool operations. This isn't safe security-wise
+// as it's attaching a sandbox of multiple containers to it, containing
+// untrusted data. This may be fine for client devops scenarios. In
+// safe mode, a unique utility VM is instantiated for all service VM tool
+// operations. The downside of safe-mode is that operations are slower as
+// a new service utility VM has to be started and torn-down when needed.
+//
+// To enable global mode, run with --storage-opt lcowglobalmode=1
```

Comment:
actually, can we change this option to `--storage-opt lcow.globalmode=1`?

Storage-driver specific options use a `<driver>.` prefix (see https://github.com/docker/cli/blob/ff2552f7a1073834e79793aeb5dc5a4e486956a4/docs/reference/commandline/dockerd.md#options-per-storage-driver)